mobile/native/src/names.rs

```diff
@@ -0,0 +1,26 @@
+pub fn get_new_name() -> String {
+    let names = petname::Petnames::default();
+    let input = names.generate_one(2, " ");
```

Comment:
🔧 I would expect it to be separated with a hyphen!

Reply:
No offense, but my name is 

<img width="299" alt="image" src="https://github.com/get10101/10101/assets/224613/86dc5bde-49d8-4f15-a532-ffde40a90f37">

which reads much better with capital letters and a space in between :D